### PR TITLE
blog: HTTP/2 Smarter At Scale

### PR DESCRIPTION
**Pull request by jadekler**
Add `HTTP/2: Smarter At Scale` article.

This is a crosspost of https://www.cncf.io/blog/2018/07/03/http-2-smarter-at-scale/.

### CODE REVIEW

**Comment by jadekler** [comment]
ps: Is there a way to check this out locally or something beforehand?

**Comment by carl-mastrangelo** [comment]
@jadekler alas,, there is not.  You can try running your own jekyll / liquid instance locally.  @LisaFC knows more.

**Comment by hsaliak** [comment]
Already reviewed as part of the CNCF post. Merge when you think is appropriate.

**Comment by jadekler** [comment]
I'm not able to merge haha. Could you add me to the org, or repo?

**Comment by nicolasnoble** [comment]
I can merge for you - is there a specific time you'd want this to happen or ASAP is good ?

**Comment by jadekler** [comment]
@nicolasnoble Actually, come to think on it, I should probably do it on monday since I suspect the images may be gargantuan sized on the blog :)

File: _posts/2018-07-13-http2_smarter_at_scale.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+HTTP/2 solves this problem by providing a flow control mechanism as part of the stream specification. Flow control is used to limit the amount of outstanding data on a per-stream (and per-connection) basis. It operates as a credit system in which the receiver allocates a certain “budget” and the sender “spends” that budget. More specifically, the receiver allocates some buffer size (the “budget”) and the sender fills (“spends”) the buffer by sending data.The receiver advertises to the sender additional buffer as it is made available, using special-purpose WINDOW_UPDATE frames. When the receiver stops advertising additional buffer, the sender must stop sending messages when the buffer (its “budget”) is exhausted.
```
[the remark this answers is not in the document]
Comment (jadekler) [reply _posts/2018-07-13-http2_smarter_at_scale.md]: done

**Comment by carl-mastrangelo** [comment]
@jadekler We usually post on Tuesday morning.   I'll merge it then.

**Comment by carl-mastrangelo** [comment]
@jadekler merged, Thanks!

**Comment by nicolasnoble** [comment]
Erm, this seems broken on live.

https://grpc.io/blog/

https://grpc.io/2018/07/24/2018-07-13-http2_smarter_at_scale.html

**Pull request merged**